src/app/workflow/dag/state/dag.service.ts

```diff
@@ -326,6 +329,10 @@ export class DagService {
     };
     const destroy = () => {
       try {
+        const tooltips = document.getElementsByClassName(dagTooltipAttribute);
+        for (var i = 0; i < tooltips.length; i++) {
```

Comment:
I tried to use the `forEach` method butI couldn't since `document.getElementsByClassName(dagTooltipAttribute)` is a `HTMLCollection` and not an `Array`. I'll try to find another alternative.

Reply:
Changed `var` to `let`